merging:
  number_of_velodynes: 3
  b_use_random_filter: true
  b_use_radius_filter: false
  decimate_percentage: 0.8
  radius: 0.3
  radius_knn: 2

  ext_lidar1: [1, 0, 0, 0,
               0, 1, 0, 0,
               0, 0, 1, 0,
               0, 0, 0, 1]

  ext_lidar2: [-0.0309817, -0.999501, -0.00619944, -1.90946,
               0.995198, -0.030271, -0.093082, 0.877353,
               0.0928478, -0.00985351, 0.995639, 0.0198776,
               0, 0, 0, 1]

  ext_lidar3: [0.0135122, 0.999164, -0.0385849, -1.81837,
               -0.996531, 0.0166255, 0.0815407, -0.975421,
               0.0821137, 0.0373493, 0.995923, 0.0141756,
               0, 0, 0, 1]

